### DISCUSSION TITLE
Security Onion Multiple Master Nodes

### DISCUSSION THREAD

**Discussion by ryguy-vt** [Security-Onion-Solutions/securityonion#1751] [Unsupported Versions] [answered]
Could the security onion architecture allow for two master nodes? In case of a failure scenario we would like redundancy on our systems.

Similarly, are we able to send logs, such as system logs, from the master node to a different server?

**Comment by joeb1kenobe** [comment]
I have a setup that is some what similar to what you describe. I have a manager node on a Hyper-V cluster. That uses Hyper-V replication to replicate the node to our hotsite environment. When time comes, we switch off the prod manager, and on the hotsite manager. 

In our environment, we had to change the IP address on the server manually after it failed over.

**Comment by TOoSmOotH** [comment, marked as answer]
We have an open issue for this: 
https://github.com/Security-Onion-Solutions/securityonion/issues/80

**Reply by rwaight** [reply]
@TOoSmOotH does https://github.com/Security-Onion-Solutions/securityonion/issues/80 also include [high-availability with Salt](https://docs.saltproject.io/en/latest/topics/highavailability/index.html)?

The reason I ask is that I am testing an SO environment with over 18 nodes and have been running into a `unhealthy manager` prompt when attempting to add more search nodes to my Security Onion grid.  There are probably other issues around why I see that message, but just curious if a consideration for https://github.com/Security-Onion-Solutions/securityonion/issues/80 includes HA for Salt.

**Reply by TOoSmOotH** [reply]
@rwaight This is a different issue specific to saltstack vs elastic
